docs/react/introduce.en-US.md

```diff
@@ -0,0 +1,95 @@
+---
+order: 0
+title: Ant Design of React
+---
+
+antd is a set of React components which follow Ant Design specification. It is designed to develop RIA sucn as dashboard to serve enterprise.
```

Comment:
antd is a set of React components which follow the Ant Design specification. It is designed to help developing RIA such as dashboards or other enterprise-like complex UI needs.